app/controllers/api/v1/event_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::EventController < ApiController
+  def index
+    render json: Event.all
```

Comment:
true, but @bitTal will still work on this, and we need to reevaluate if we can still use the pagination gem or not